runtime/onert/core/src/compiler/train/TrainableOperationConverter.cc

```diff
@@ -68,7 +68,13 @@ void TrainableOperationConverter::visit(const ir::operation::FullyConnected &nod
 
 void TrainableOperationConverter::visit(const ir::operation::Loss &node)
 {
-  _return_op = std::make_unique<ir::train::operation::Loss>(node, _training_info->lossInfo());
+  const auto &y_pred_index = node.getInputs().at(ir::operation::Loss::Input::Y_PRED);
+  const auto &y_pred = _tgraph.operands().at(y_pred_index);
+  const auto &y_pred_node = _tgraph.operations().at(y_pred.getDef());
+  const auto y_pred_op_code = y_pred_node.opcode();
+
+  _return_op =
+    std::make_unique<ir::train::operation::Loss>(node, _training_info->lossInfo(), y_pred_op_code);
```

Comment:
Just for curiosity, (not related to this PR) 

Is there a case that `ir::operation::Loss` converted into `ir::train::operation::Loss`?  
I thought there wasn't a `Loss` operation for inference, So I wondered where `ir::operation::Loss` is generated.

Reply:
There is no case to convert it because `ir::operation::Loss` is generated only in `LossInsertionPass`. It seems that this method will be called again in the future. So, I think it's OK to remove this method.

Reply:
@zetwhite Would you like to remove this method that may be unused in the future?

Reply:
Even if it's not being used, I'm fine with leaving it as-is. I'd like to follow your decision (removing it or not).

Reply:
I create a related PR #14168